R/draws.R

```diff
@@ -1,10 +1,112 @@
-#' TODO
+#' @title Fit the base imputation model and get parameter estimates
+#'
+#' @description Assuming MAR, `draws` fits the base imputation model to the observed data
+#' according to the given multiple imputation methodology and returns the parameter
+#' estimates from the fit.
 #'
 #' @name draws
-#' @param data TODO
-#' @param data_ice TODO
-#' @param vars TODO
-#' @param method TODO
+#' @param data A `data.frame` containing the data to be used in the model.
+#' Must contain the outcome variable. See details.
```

Comment:
and any covariates ?

Reply:
I wanted to explicitly write that it should contains also the outcome variable. But yes, I think the necessary variables are subject ids, group, visit, outcome. Correct?

Reply:
I will just omit "Must contain the outcome variable." since everything is described in the details

Reply:
Sure